test/builtin/numbers/test_randomnumbers.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for mathics.builtins.numbers.randomnumbers
+"""
+
+import pytest
+from test.helper import check_evaluation
+
+
+@pytest.mark.parametrize(
+    ("str_expr", "str_expected"),
+    [
+        ("SeedRandom[42]; RandomSample[{a, b, c, d}]", "{b, d, a, c}"),
+        (
+            "SeedRandom[42]; RandomSample[{a, b, c, d, e, f, g, h}, 7]",
+            "{b, f, a, h, c, e, d}",
```

Comment:
I am kind of curious. If SeedRandom is a different number, do we get a different order?

Reply:
In my tests, yes, changing the SeedRandom value results in a different order:
```Mathematica
In[1]:= SeedRandom[42]; RandomSample[{a, b, c, d}]
Out[1]= {b, d, a, c}

In[2]:= SeedRandom[137]; RandomSample[{a, b, c, d}]
Out[2]= {a, b, d, c}

In[3]:= SeedRandom[137]; RandomSample[{a, b, c, d}]
Out[3]= {a, b, d, c}

In[4]:= SeedRandom[962]; RandomSample[{a, b, c, d}]
Out[4]= {d, a, c, b}
```

And I also assume that changing the random number generator will also change the order.